exporters/compression.py

```diff
@@ -39,7 +39,7 @@ def get_compress_file(compression_format):
     from bz2file import BZ2File
 except ImportError:
     import logging
-    logging.warning('bz2file module not found. BZ2 compression disabled. To enable it please '
-                    'install bz2file package or install lazy_requirements.txt')
+    logging.info('bz2file module not found. BZ2 compression disabled. To enable it please '
+                 'install bz2file package')
```

Comment:
What about simply: `logging.info('Install bz2file to enable BZ2 compression.')` ?